2018-westphal-antpc.tex

```diff
@@ -120,48 +115,48 @@ \subsection{Electroreduction}
 
-Electroreduction produces the majority of Sr and Cs waste compared to electrowinning. This stream has considerable decay 
-signature proportional to the efficiency and size of the feed batch \cite{Borrelli_2017,flowsheet_1998}.
+This stream has considerable decay signature proportional to the efficiency and size of the feed batch \cite{Borrelli_2017,flowsheet_1998}.
 
 \subsection{Electrorefining}
 
-The uranium and salt mixture from reduction is fed into an anode basket suspended in a graphite cathode. A LiCl-KCl eutectic is used as 
-an electrolyte above 500$^{\circ}$C \cite{flowsheet_1998,lee_korean_2011}. The uranium dissolves at the anode to recombine at 
-the cathode as metallic uranium. The waste transuranics (TRUs) and lathanides are in a soluble chloride form  while fission 
-products and cladding remain in the anode basket. Finally, actinides and fission products are removed from the cladding 
-electrochemically \cite{lee_korean_2011}.
+Once in metallic form, electrorefining electrochemically separates uranium and TRUs for fuel fabrication.
+The uranium and salt mixture from reduction is fed into an anode basket suspended in a graphite cathode. 
+A LiCl-KCl eutectic is used as an electrolyte above 500$^{\circ}$C \cite{flowsheet_1998,lee_korean_2011}. 
+Uranium dissolves at the anode to recombine at the cathode as metallic uranium. 
+Waste transuranics (TRUs) and lanthanides are in a soluble chloride form  while fission products and cladding remain in the anode
+basket. Finally, actinides and fission products are removed from the cladding electrochemically \cite{lee_korean_2011}.
 
-Lee et al. \cite{lee_advanced_2008} show that the addition of vacuum pressure to the system improves removal efficiency. 
+Lee et al. \cite{lee_advanced_2008} show decreasing system pressure improves removal efficiency. 
 Temperature, however, exhibits the opposite effect: as temperature decreases so does salt removal. This comes into effect 
-particularly depending on material choice of instrumentation and containment \cite{lee_advanced_2008}. The most limiting 
-material choice is iron because a eutectic forms between Fe and U at 725$^{\circ}$C \cite{chapman_revision_1984}. 
-In facilities where iron equipment is present temperatures are limited to 700$^{\circ}$C, and efficiency is 
-significantly hindered. Cathode arrangement and anode rotation speed also affect the collection of uranium 
-dendrites \cite{lee_advanced_2008}. The addition of a central stirrer allows mixing of uranium dendrites stuck on 
-the bottom of the vessel, improving separation efficiency and increasing throughput. 
+particularly depending on material choice of instrumentation and containment \cite{lee_advanced_2008}. 
+Iron, for example, limits operating temperature because a eutectic forms at 725$^{\circ}$C \cite{chapman_revision_1984}.
+In facilities where iron equipment is present, temperatures are limited to 700$^{\circ}$C, and efficiency is significantly hindered. 
```

Comment:
For brevity/clarity:

"and efficiency is significantly hindered" -> "hindering efficiency"